frontend/src/components/FeedResults/FeedResults.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button } from '@components/Button'
+import { Box, IconOutlineArrowNarrowUp, Text } from '@highlight-run/ui'
+import { AnimatePresence, motion } from 'framer-motion'
+import React from 'react'
+
+import * as style from './FeedResults.css'
+
+interface Props {
+	more: number
+	type: 'sessions' | 'errors' | 'logs'
+	onClick: () => void
+}
+
+export const AdditionalFeedResults = function ({ more, type, onClick }: Props) {
+	return (
+		<AnimatePresence>
+			{more > 0 ? (
+				<motion.div
+					key="AdditionalFeedResultsWrapper"
+					initial={{ opacity: 0 }}
+					animate={{ opacity: 1 }}
+					exit={{ opacity: 0 }}
+					transition={{ duration: 0.1 }}
+				>
+					<Box
+						paddingTop={type === 'logs' ? undefined : '8'}
+						px={type === 'logs' ? undefined : '8'}
+						width="full"
+						display="flex"
+						alignItems="center"
+						justifyContent="center"
+					>
+						<Button
+							kind="secondary"
+							size="small"
+							emphasis="high"
+							trackingId="SessionsFeedMore"
+							className={style.moreButton}
+							onClick={onClick}
+						>
+							<Box display="flex" alignItems="center" gap="8">
+								<IconOutlineArrowNarrowUp />
+								<Text>
+									{more}
+									{type === 'logs'
```

Comment:
Should we include the 50+ for more types than logs if we happen to hit that case?

Reply:
the logs query is paginated in such a way that we don't know how many more logs than 50 we have, hence i'm showing 50+. for sessions and errors, we have the exact count of total new sessions / errors.